### PR TITLE
alias logs cmd

### PR DESCRIPTION
**Pull request by cbron**

### CODE REVIEW

File: cli/cmd/logs/logs.go [hunk 1]
```diff
@@ -31,6 +32,10 @@ type Logs struct {
 
 // This is based on both wercker/stern and linkerd/stern implementations
 
+func (l *Logs) Customize(cmd *cli.Command) {
+	cmd.ShortName = "log"
```
Comment (StrongMonkey) [inline cli/cmd/logs/logs.go]: Wait, shouldn't that be alias

Comment (cbron) [reply cli/cmd/logs/logs.go]: https://github.com/rancher/rio/blob/master/cli/cmd/images/image.go#L47-L49

Comment (StrongMonkey) [reply cli/cmd/logs/logs.go]: ahhh
> // short name of the command. Typically one character (deprecated, use `Aliases`)

Comment (StrongMonkey) [reply cli/cmd/logs/logs.go]: I was wrong for the usage, but should be fine.

**Pull request merged**